integration-tests/metrics-v4.2.19.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
To ensure the `integration-test.sh` is working for both cases, it'd be nice to also rewrite the content of the `checkstyle-10.12.4.sh` file to sue the script.